R/tm_g_distribution.R

```diff
@@ -27,9 +27,13 @@
 #'
 #' @inherit shared_params return
 #'
-#' @examples
-#' library(teal.widgets)
-#'
+#' @examplesShinylive
+#' library(teal.modules.general)
+#' interactive <- function() TRUE
+#' {{ next_example }}
+# nolint start: line_length_linter.
+#' @examplesIf require("ggpmisc", quietly = TRUE) && require("ggpp", quietly = TRUE) && require("goftest", quietly = TRUE) && require("MASS", quietly = TRUE) && require("broom", quietly = TRUE)
```

Comment:
FYI: this line of code would be included in the application code. It's important to use only `library()` or `require()` because those two are currently shimmed by `{webr}`(*). In particular: we cannot use `requireNamespace()` (**). I created a new task for `{webr}` to add this. Until this happen - let's stick to the `require()`.

(*) shimming is overwriting in this context and `{webr}` will do install (if needed) & load every time you run these
(**) Actually we can but then we would have to include additional `library()` or `require()` in the shinyliveexample section which seems to be redundant.